web/decap-config/collections/01-business-formation.yml

```diff
@@ -0,0 +1,1175 @@
+collections:
```

Comment:
Are we ok with how we are naming this file? I added two digits as a prefix, this ensures the collections appear in the same order in the collections list in the cms.